_posts/commands/2020-08-24-mom_prog_tele.md

```diff
@@ -0,0 +1,10 @@
+---
+title: mom_prog_tele
+category: command
+tags:
+  - zones
+  - triggers
```

Comment:
Just have the "teleport" tag here; triggers one is used for documenting triggers themselves and zones is more for documenting the zoning commands/cvars.